scripts/verify-gopkg.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+TOML=$( grep -P '( |^)name = ".*"' Gopkg.toml | tr -d ' ' | sort )
+LOCK=$( grep -P '( |^)name = ".*"' Gopkg.lock | tr -d ' ' | sort )
+
+# NOTE(ericchiang): For some reason you need this "export" here for bash reasons
+# I don't understand.
+export DIFF=$( diff <(echo "$TOML") <(echo "$LOCK") )
```

Comment:
Given you comment, and a quick test, did you ensure this works in the failure case?

You don't wrap the value in quotes, and `DIFF=` and `export DIFF=` work differently if the values have spaces, at least in my quick little experiment it seemed to.

It seems like maybe `export` is papering over the fact that your value has whitespace and allowing for false positives?

Reply:
might need a `-q` in there as well to only print when files differ

Reply:
I confirmed that this works when the files are out of sync.

If I comment out the `export` and modify one of the Gopkg files I can replicate locally that `DIFF` becomes set to the empty string no matter what.

I can add some quotes, though I tested that and it didn't work to fix this problem. @thorfour and I were staring at this for a while and don't have any good answer here.